inference-engine/tests/unit/vpu/middleend_tests/passes_tests/adjust_data_batch_tests.cpp

```diff
@@ -0,0 +1,389 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vpu/middleend/allocator/allocator.hpp>
+#include <vpu/stages/mx_stage.hpp>
+#include <vpu/utils/numeric.hpp>
+
+#include "graph_transformer_tests.hpp"
+
+using namespace vpu;
+
+class VPU_AdjustDataBatchTest_New_Inf : public GraphTransformerTest {
+public:
+    int batchSize;
+
+    TestModel testModel;
+
+    void SetUp() override {
+        ASSERT_NO_FATAL_FAILURE(GraphTransformerTest::SetUp());
+
+        ASSERT_NO_FATAL_FAILURE(InitCompileEnv());
+
+        batchSize = 4;
+
+        DataDesc dataDesc(DataType::FP16, DimsOrder::NCHW, {16, 16, 3, batchSize});
+        testModel = CreateTestModel(dataDesc);
+    }
+
+    void RunPass() {
+        PassSet pipeline;
+        pipeline.addPass(passManager->dumpModel("initial"));
+        pipeline.addPass(passManager->adjustDataBatch());
+        pipeline.addPass(passManager->dumpModel("adjustDataBatch"));
```

Comment:
same here.

Reply:
Actually, it is normal practice to dumpModel in unit tests. I guess we can leave it.